pkg/checkoutinfo/checkoutinfo.go

```diff
@@ -0,0 +1,114 @@
+package checkoutinfo
```

Comment:
I'd expect to see project info mediated here that is reflected in the URL. So org, name, namespace and branch.

Basically those fields should all be handled through this package, and the project/projectfile package can still work with them through the legacy pattern, same as we're doing with the commitID.

Reply:
Right. I was hoping we could eliminate `checkoutinfo.SetCommitID()`, but buildscripts are still hidden behind a config wall. Until we remove that wall, we'll have the SetCommitID -> update activestate.yaml, update build script -> update activestate.yaml flow.